src/main/java/com/database/operations/CompanyRepository.java

```diff
@@ -0,0 +1,85 @@
+package com.database.operations;
+
+import com.database.data.Company;
+import com.database.data.DBCloseConnect;
+import com.database.data.JdbcConnectionUtil;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CompanyRepository {
+
+    // Qwery1 - SELECT_ALL, Qwery2 - SELECT_ID , Qwery3 - INSERT;  Qwery4 - DELETE;  Qwery5 - UPDATE;
```

Comment:
Very bad practice to use a badly thoughtful name and after that add comment to explain what is it.
So, you should fix it.
Also naming style is not right https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names